src/scripts/common.js

```diff
@@ -45,8 +45,11 @@ export const sleep = function(amount) {
 }
 
 export function toMicroDenom(denom) {
-  if (denom === "ATOM") {
-    return "uatom"
+  const lookup = {
+    ATOM: "uatom",
```

Comment:
will we need to add this for every network?

Reply:
I tried using `substr` and just removing the first character, the problem are the "other" tokens... like `ukr` in Terra and all the others in e-Money... they also go through this function.

So right now I don't see any other solution :woman_shrugging:

Reply:
we should probably plan to store this in the db / api.

we want to get to a place where we don't need to update lunie frontend to add new networks.

(i think)

Reply:
Yes, that would be the ideal

Reply:
#overengineering

Reply:
hahaha. One thing is the ideal and another one the raw reality :laughing: